config/filter.d/postfix.conf

```diff
@@ -17,7 +17,7 @@ _pref = [A-Z]{4}
 prefregex = ^%(__prefix_line)s<mdpr-<mode>> <F-CONTENT>.+</F-CONTENT>$
 
 # Extended RE for normal mode to match reject by unknown users or undeliverable address, can be set to empty to avoid this:
-exre-user = |[Uu](?:ser unknown|ndeliverable address)
+exre-user = |(?:[Uu]ser unknown|[Uu]ndeliverable address)
```

Comment:
Don't think this'd be a part of spelling/typos fixing branch...
Moreover why it must be changed at all? Because otherwise "stupid" spelling tool bothers on `ndeliverable`?

Reply:
yeah -- I made it into a separate commit even for that reason so we get a clear dedicated description. It is the `ser` which is a possible typo for `set` (and may be smth else).  So indeed, it is a code change to please `codespell`. But you are the boss -- if you like, I can just ignore `ser` (and thus we could let some typos in) globally and drop that commit.

Reply:
Ignore globally or is there some pragma doing the job for single line? (Just to know how it's possible)

Reply:
there is https://github.com/codespell-project/codespell/issues/1212 which is still open but as a comment there [says](https://github.com/codespell-project/codespell/issues/1212#issuecomment-1744768533) it is trivial to workaround by tuning regex-skip and defining custom pragma. So let's code for a line with `codespell-ignore` in it to be ignored. I will add that now and drop that commit.

Reply:
I have done that, will mark resolved